src/index.js

```diff
@@ -0,0 +1,91 @@
+import './style.css';
+import TodoItem from './modules/properties';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Project from './modules/addProject';
+import ProjectForm from './modules/newProject';
+import Projects from './modules/project';
+import addToDoForm from './modules/addToDo';
+import items from './modules/toDoItems';
+import Store from './modules/localStorage';
+
+window.addEventListener('DOMContentLoaded', ProjectForm.hideForm);
+ProjectForm.viewAllProject();
+
+// selectors
+const addSign = document.querySelector('#add-project');
+const form = document.getElementById('form');
+const todobtn = document.getElementById('add-todo');
+const cancelProjectFormBtn = document.getElementById('cancelproject-form');
+
+// functions
+
+const clearProjectList = (container) => {
+  document.querySelectorAll(container).forEach(project => project.remove());
+};
+
+const filterProject = (project) => {
+  let find = -1;
+  for (let i = 0; i < Projects.length; i += 1) {
+    if (project.title === Projects[i].title) {
+      find = 1;
+      break;
+    }
+  }
+  if (find === -1) {
+    Projects.push(project);
+    Store.addProject(project);
+  } else {
+    alert('Project already exists!');
+  }
+};
+
+
+// event listners
+
+addSign.addEventListener('click', () => {
+  ProjectForm.showForm();
+});
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('project-title').value.toUpperCase();
+  const project = new Project(title);
```

Comment:
## Add a `default` project to the page.
- [ ] The app currently doesn't have a default project. kindly add this so that users can be able to use the default project in creating new todos when there won't be a need of creating a new project.